### PR TITLE
Mark 4.16.0 release

### PR DESCRIPTION
**Pull request by mzyromski-atlassian**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,13 +24,17 @@ Adding a requirement of a major version of a dependency is breaking a contract.
...
+## [4.16.0] - 2020-06-02
+[4.16.0]: https://github.com/atlassian/infrastructure/compare/release-4.15.1...release-4.16.0
 
 ### Fixed
-- Spurious apt warnings have been eliminated ([JPERF-642](https://ecosystem.atlassian.net/browse/JPERF-642))
+- Eliminate spurious apt warnings. Fix [JPERF-642](https://ecosystem.atlassian.net/browse/JPERF-642).
 
 # Added
-- a new, recommended method to add new repositories to Ubuntu: com.atlassian.performance.tools.infrastructure.api.os.Ubuntu.addRepository(com.atlassian.performance.tools.ssh.api.SshConnection, java.lang.String, java.lang.String)
+- Add a new recommended method to add new repositories to Ubuntu: `com.atlassian.performance.tools.infrastructure.api.os.Ubuntu.addRepository(com.atlassian.performance.tools.ssh.api.SshConnection, java.lang.String, java.lang.String)`.
```
Comment (wyrzyk) [inline CHANGELOG.md:37]: I'm not sure if we need such level of details in a changelog 🤔

Comment (mzyromski-atlassian) [reply CHANGELOG.md:37]: Yes, I was wondering about it too. I guess we don't, but I guess it's not a problem to have it.

**Comment by mzyromski-atlassian** [comment]
Artifacts: https://packages.atlassian.com/maven-public/com/atlassian/performance/tools/infrastructure/4.16.0/

File: CHANGELOG.md (hunk 1 above)
Comment (dagguh) [reply CHANGELOG.md:37]: Changelogs are for consumers. They search the page or read a given version.
> - Add `Ubuntu.addRepository`.

**Pull request merged**